content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -508,7 +508,83 @@ flags:
 
 {{< /highlight >}}
 
+## upgrade_release workflow
 
+This workflow provides the ability to upgrade Helm release created with Helm plugin.
+
+**Parameters:**
+
+* `node_instance_id` - Node instance ID of `cloudify.nodes.helm.Release` node type. This node instance represents Helm release that will be upgraded.
+
+**required: true**
+
+* `chart` - The chart to upgrade the release with. The chart argument can be either: a chart reference('example/mariadb'), path to packaged chart, or a fully qualified URL.
+
+**required: true**
+
+* `flags` - Flags to add for `helm upgrade` command. The format is the same as "flags" property.
+
+* `set_values` -  List of variables names and values to set. For example:
+
+{{< highlight  yaml  >}}      
+    - name: x 
+      value: y
+    - name: a
+      value: b
+{{< /highlight >}} 
+
+Equals adding `--set x=y --set a=b` to helm command.
+
+* `values_file:` - Path to values file.
+
+### Example of using upgrade_release workflow
+
+Assuming the release node type is :
+
+{{< highlight  yaml  >}}
+
+node_templates:
+
+  release:
+    type: cloudify.nodes.helm.Release
+    properties:
+      client_config:
+        configuration:
+          file_content: {get_secret: kube_config }
+        authentication:
+          gcp_service_account: { get_secret: gcp_credentials }
+      resource_config:
+        name: "myrelease"
+        chart: bitnami/postgresql
+    relationships:
+      - target: helm_install
+        type: cloudify.helm.relationships.run_on_host
+      - target: repo
+        type: cloudify.relationships.depends_on
+
+{{< /highlight >}}
+
+`upgrade_release` can triggered this way:
+
+`cfy executions start upgrade_release -d release-upgrade-test -p node_instance_id=release_8uwvap -p ./inputs.yaml`
+
+Where `inputs.yaml` contains:
+
```

Comment:
Maybe you want to document what the runtime properties are and what the output might look like

Reply:
Done